allensdk/test/api/cloud_cache/test_static_local_cache.py

```diff
@@ -0,0 +1,149 @@
+                f"/project_metadata/{meta_fname}"
+            ),
+            "version_id": "test_placeholder",
+            "file_hash": file_hash_from_path(meta_save_path)
+        }
+
+    # Create data files and manifest entries
+    mock_data_dir = mock_project_dir / "project_data"
+    mock_data_dir.mkdir()
+
+    manifest_data_entries = dict()
+    for file_fname, file_contents in data_files_contents:
+        file_save_path = mock_data_dir / file_fname
+        with file_save_path.open('w') as f:
+            f.write(file_contents)
+
+        manifest_data_entries[file_fname.rstrip(".nwb")] = {
+            "url": (
+                f"http://{project_name}.s3.amazonaws.com/{project_name}"
+                f"/project_data/{file_fname}"
+            ),
+            "version_id": "test_placeholder",
+            "file_hash": file_hash_from_path(file_save_path)
+        }
+
+    # Create manifest dir and manifest
+    mock_manifests_dir = mock_project_dir / "manifests"
+    mock_manifests_dir.mkdir()
+    manifest_fname = f"test_manifest_v{dataset_version}.json"
+    manifest_path = mock_manifests_dir / manifest_fname
+
+    manifest_contents = {
+        "project_name": project_name,
+        "manifest_version": dataset_version,
+        "data_pipeline": [
+            {
+                "name": "AllenSDK",
+                "version": "2.11.0",
+                "comment": "This is a test entry. NOT REAL."
+            }
+        ],
+        "metadata_file_id_column_name": metadata_file_id_column_name,
+        "metadata_files": manifest_meta_entries,
+        "data_files": manifest_data_entries
+    }
+
+    with manifest_path.open('w') as f:
+        json.dump(manifest_contents, f, indent=4)
+
+    expected = {
+        "expected_metadata": metadata_files_contents,
+        "expected_data": data_files_contents
+    }
+
+    return mock_mounted_base_dir, project_name, expected
+
+
+@pytest.mark.parametrize(
+    "mounted_s3_dataset_fixture",
+    [
+        {"project_name": "visual-behavior-ophys"}
+    ],
+    indirect=["mounted_s3_dataset_fixture"]
+)
+def test_static_local_cache_access(mounted_s3_dataset_fixture):
+    local_static_cache_dir, proj_name, expected = mounted_s3_dataset_fixture
+
+    cache = StaticLocalCache(local_static_cache_dir, proj_name)
+    cache.load_last_manifest()
+
+    for exp_meta_fname, exp_meta_contents in expected["expected_metadata"]:
+        exp_df = pd.DataFrame(exp_meta_contents)
+        obt_df_path = cache.metadata_path(exp_meta_fname.rstrip(".csv"))
+        obt_df = pd.read_csv(obt_df_path["local_path"])
+        pd.testing.assert_frame_equal(exp_df, obt_df)
+
+    for exp_data_fname, exp_data_contents in expected["expected_data"]:
+        obt_data_path = cache.data_path(exp_data_fname.rstrip(".nwb"))
+        with open(obt_data_path["local_path"], "r") as f:
+            obt_data = f.read()
+        assert exp_data_contents == obt_data
```

Comment:
I would feel more comfortable if there was also a unit test for the `BehaviorProjectCloudApi` using this mocked static cache

(see `allensdk/test/brain_observatory/behavior/behavior_project_cache/test_behavior_project_cloud_api.p`)

I acknowledge that is probably redundant. Feel free to ignore this request.

Reply:
I've added a test to address the changes in the behavior_project_cloud_api. Just need to check that the right class is called.